rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/asset_search/action.py

```diff
@@ -0,0 +1,72 @@
+import ast
+
+import insightconnect_plugin_runtime
+from .schema import AssetSearchInput, AssetSearchOutput, Input, Output, Component
+
+
+# Custom imports below
+
+
+def format_response(resources: [], params={}):
+    assets = list()
+    ip = params.get(Input.IP)
+    hostname = params.get(Input.HOSTNAME)
+    for page in range(len(resources)):
+        current = str(resources[page])
```

Comment:
```suggestion
    for page in resources:
```
You can replace current with `page`
I might be missing something but I think this is what you're trying to do.